### ISSUE TITLE
Question on reconstruction loss multiplier(s)

### ISSUE THREAD

**Issue by adizhol** [tolstikhin/wae#6]
Hello Sir,

I saw that in the reconstruction_loss function (wae.py) you multiply L2, L2_2 and L1 losses with different kind of values, but I haven't read anything about this in the paper.
Can you please explain?

Thank you,
Adi Zholkover

**Comment by tolstikhin** [comment]
Dear Adi,

thanks for the question! This is done in the code to confuse the reader.... Kidding! :-) The thing is this is a legacy from the older code, where we did some hyperparam tuning and there was no easy way to merge the code, so we ended up hard-coding these constants. They only affect the lambda / learning_rate.

**Comment by adizhol** [comment]
Okay, thank you :)
